scripts/benchmark_all.sh

```diff
@@ -157,14 +164,13 @@ do
       case "${config}" in
         "st") echo -n "single-threaded, SF 10.0" ;;
         "st_s01") echo -n "single-threaded, SF 0.01" ;;
-        "st_s1") echo -n "single-threaded, SF 1.0" ;;
-        "mt_clustered") echo -n "multi-threaded, clustering config 'Pruning', SF 10.0" ;;
-        "mt") echo -n "multi-threaded (${num_mt_clients} clients), SF 10.0" ;;
+        "mt") echo -n "multi-threaded, shuffled, ${num_phy_cores} clients, ${num_phy_cores} clients,  SF 10.0" ;;
+        "mt_ordered") echo -n "multi-threaded, ordered, ${num_phy_cores} clients, ${num_phy_cores} clients,  SF 10.0" ;;
```

Comment:
If we want to execute a run with the case _single client, multiple cores_ configuration, we should adapt the description about the number of clients here accordingly.